src/inventory/gilded_rose.spec.js

```diff
@@ -10,25 +10,24 @@ describe('updateQuality', () => {
     });
     // ----Quality deprecates by double for sell_in < 0----//
     it('deprecates quality by double when sell_in<0', () => {
-      const standardItem = new Item('Haunted Shoe', 0, 5);
+      const standardItem = new Item('Haunted Shoe', 0, 7);
       updateQuality([standardItem]);
-      expect(standardItem.quality).toBe(3);
+      expect(standardItem.quality).toBe(5);
     });
   });
 
   // ----Aged Brie Quality increases with age----//
   // increase sell_in to 3 and quality to 1 --passes
   describe('when updating Aged Brie', () => {
     it('increase quality of Aged Brie with age', () => {
-      const standardItem = new Item('Aged Brie', 3, 0);
+      const standardItem = new Item('Aged Brie', 3, 4);
       updateQuality([standardItem]);
-      expect(standardItem.quality).toBe(1);
+      expect(standardItem.quality).toBe(5);
```

Comment:
Was there reasoning behind changing the numbers used in these tests? My only concern is that is I switch this back to 0 and 1, the test no longer passes so we'd want to make sure that that condition passes too!